client/src/containers/LoginPage.tsx

```diff
@@ -3,11 +3,31 @@ import { t } from "@lingui/macro";
 
 import StandalonePage from "components/StandalonePage";
 import Login from "components/Login";
+import { useContext, useEffect, useState } from "react";
+import { UserContext } from "components/UserContext";
+import { LocaleRedirect } from "i18n";
+import { createWhoOwnsWhatRoutePaths } from "routes";
+import { FixedLoadingLabel } from "components/Loader";
 
 const LoginPage = withI18n()((props: withI18nProps) => {
   const { i18n } = props;
+  const { account } = createWhoOwnsWhatRoutePaths();
+  const userContext = useContext(UserContext);
+  const fetchingUser = !userContext?.user;
+  const isLoggedIn = !!userContext?.user?.email;
```

Comment:
✨so much clearer✨

we should consider refactoring other pages that have this similar user context logic so we aren't doing `!!userContext?.user?.email` everywhere, for readability's sake

Reply:
oh yeah good idea, I'll make a ticket so we don't forget.